src/cli.ts

```diff
@@ -137,6 +137,6 @@ function generateTypedoc() {
         './dist/docs/api',
         './src/client/tsconfig.json'
     ].map(filePath => path.resolve(filePath));
-    const cmd = `"${typedocCmd}" --name "OpenFin ${config.SERVICE_NAME}" --theme "${themeDir}" --out "${outDir}" --excludeNotExported --excludePrivate --excludeProtected --hideGenerator --tsconfig "${tsConfig}" --readme none`; // eslint-disable-line
+    const cmd = `"${typedocCmd}" --name "OpenFin ${config.SERVICE_NAME.charAt(0).toUpperCase() + config.SERVICE_NAME.slice(1)}" --theme "${themeDir}" --out "${outDir}" --excludeNotExported --excludePrivate --excludeProtected --hideGenerator --tsconfig "${tsConfig}" --readme none`; // eslint-disable-line
```

Comment:
Sneaking this in.  Opting for this instead of CSS in case of modules not expecting to be capitalized.  This route will only capitalize the docs name